File: swimming-contest-software/Model/src/main/java/model/protocol/AddSwimmerRequest.java
package model.protocol;

import model.domain.dtos.SwimmerDTO;

public class AddSwimmerRequest implements Request {

    private final SwimmerDTO swimmerDTO;

    public AddSwimmerRequest(SwimmerDTO swimmerDTO) {
        this.swimmerDTO = swimmerDTO;
    }

    public SwimmerDTO getSwimmerDTO() {
        return swimmerDTO;
    }
}

File: Model/Protocol/Requests/AddSwimmerRequest.cs
using System;
using Model.Domain.DTOs;

namespace Model.Protocol.Requests;

[Serializable]
public class AddSwimmerRequest : IRequest
{
    public AddSwimmerRequest(SwimmerDTO swimmerDto)
    {
        SwimmerDTO = swimmerDto;
    }

    public SwimmerDTO SwimmerDTO { get; }
}